util/redisutil/redis_coordinator.go

```diff
@@ -63,7 +65,29 @@ func (c *RedisCoordinator) RecommendSequencerWantingLockout(ctx context.Context)
 		}
 		return url, nil
 	}
-	log.Error("no sequencer appears to want the lockout on redis", "priorities", prioritiesString)
+
+	// If we hit this line, it means no sequencer is currently wanting the lockout from Redis.
+	// A log will be emitted at different levels depending on how long it has been since the first error was logged.
+	// At first, the log will be at the debug level, but if it persists for more than 10 seconds, it will be logged at the warn level.
+	// If it persists for more than 20 seconds, it will be logged at the error level.
+	logMessage := func(level func(msg string, ctx ...interface{})) {
+		args := []interface{}{"priorities", prioritiesString}
+		level("no sequencer appears to want the lockout on redis", args...)
+	}
+
+	if c.firstSequencerWantingLockoutErrorTime.IsZero() {
+		c.firstSequencerWantingLockoutErrorTime = time.Now()
+		logMessage(log.Debug)
+	} else {
+		elapsedTime := time.Since(c.firstSequencerWantingLockoutErrorTime)
+		if elapsedTime > 20*time.Second {
+			logMessage(log.Error)
```

Comment:
These logMessage calls gets reached several times per second, how about limiting log frequency to once per 5 seconds or so?